src/test/java/pageComponents/menu/OrderSummaryPageComponent.java

```diff
@@ -1,25 +1,31 @@
 package pageComponents.menu;
 
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
+import java.time.Duration;
 import java.util.List;
 
 public class OrderSummaryPageComponent {
-
+    private WebDriverWait wait;
+    private WebDriver driver;
     public OrderSummaryPageComponent(WebDriver driver)  {
         PageFactory.initElements(driver, this);
     }
 
-    @FindBy(xpath = "//table//tbody//tr")
+    @FindBy(xpath = "//table//tbody//tr[1]")
     private List<WebElement> itemsInOrder;
```

Comment:
should it return a list or one WebElement? what do you expect to get?

Reply:
I expect to get a list of WebElements but only in the first field, I need it for my tests.